controllers/reconcile.go

```diff
@@ -744,7 +822,7 @@ func (r *FrontendReconciliation) createOrUpdateCacheBustJob() error {
 
 	j.Spec.Template.ObjectMeta.SetAnnotations(annotations)
 
-	errr := r.populateCacheBustContainer(j)
+	errr := populateContainer(j)
```

Comment:
Should this function still be a part of the frontend reconciliation object?

Reply:
It is part of that object. `populateContainer` is a parameter of this `createOrUpdateJob` function. During function call, I am still passing `r.populateCacheBustContainer` for the `populateContainer`.